src/app.ts

```diff
@@ -24,6 +26,17 @@ const csvFiles = [
   },
 ];
 
+csv(fs.readFileSync(`./csv/fips.csv`), { columns: true }, (err, output) => {
```

Comment:
what's this fips for? If it's just about getting the state for each row, we don't really need this. With the first column we already know the state name.

Reply:
sure moved it